src/main/java/com/yegor256/xsline/StLambdaQuiet.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.yegor256.xsline;
+
+import com.jcabi.xml.XML;
+import java.util.function.Supplier;
+
+/**
+ * Same as {@link StLambdaQuiet}, but with catching all checked exceptions.
+ *
+ * <p>This decorator may be useful if you don't want to create a new
+ * class for your shift, but just want a simple piece of code to
+ * do the transformation without catching exceptions.</p>
+ *
+ * @since 0.13.0
+ * @checkstyle IllegalCatchCheck (500 lines)
+ */
+public final class StLambdaQuiet implements Shift {
+
+    /**
+     * The UID.
+     */
+    private final Supplier<String> name;
+
+    /**
+     * The function.
+     */
+    private final BiFuncChecked<Integer, XML, XML> lambda;
+
+    /**
+     * Ctor.
+     * @param fun The function
+     * @since 0.13.0
+     */
+    @SuppressWarnings({"PMD.AvoidRethrowingException", "PMD.AvoidCatchingGenericException"})
+    public StLambdaQuiet(final FuncChecked<XML, XML> fun) {
+        this((integer, xml) -> {
+            try {
```

Comment:
@MikhailLipanin you don't need to catch exceptions here. they will be caught in `apply()`. here just let them float through.

Reply:
@yegor256 , but it is a Ctor, so when we make smth like that:
```
new StLambdaQuiet(
    (xml) -> {
        // place, where can occure an Exception
    }
);
```
We won't catch them, since it is just a `Function` as an argument, inside of which there is a functional method `apply`, which only `throws` an exception

Reply:
I mean, that why we process only `BiFunction.apply()`, but not `Function.apply()` ?

Reply:
I understood =)